packages/server/src/api/controllers/row/ExternalRequest.ts

```diff
@@ -629,10 +646,7 @@ export class ExternalRequest {
    * Creating the specific list of fields that we desire, and excluding the ones that are no use to us
    * is more performant and has the added benefit of protecting against this scenario.
    */
-  buildFields(
-    table: Table,
-    includeRelations: IncludeRelationship = IncludeRelationship.INCLUDE
-  ) {
+  buildFields(table: Table, includeRelations: boolean) {
```

Comment:
Any reason this is a boolean now?

Reply:
I think it's much more verbose. The IncludeRelationships is just an enum, and getting a field `IncludeRelationships` I would expect either true or false. I even found some possible bug where the code was checking `if(includeRelationships)`, treating it as a boolean

Reply:
Just one minor comment about this - I understand why boolean might be preferred to enum here, but can we use an object specifier for it as well? There might be some older code that doesn't use object specifiers, but we should strive to remove as many of them as possible.